Assets/Scripts/Actors/ActorComponents/ActorResources.cs

```diff
@@ -94,4 +154,19 @@ void SpawnResourceObject()
 			WadeUtils.ResetTransform(heldResource.transform, true);
 		}
 	}
+
+	void OnTriggerEnter(Collider other)
+	{
+		resource resourceComponent = other.gameObject.GetComponent<resource>();
+		if (resourceComponent && !resourceComponent.used)
+		{
+			resourceComponent.used = true;
+
+			Debug.Log(other.gameObject.name);
```

Comment:
> ActorResources.cs:165

I'd like to limit what debug logging goes into master. Things that give generally useful output are okay, but things that only provide information used in developing one subsystem should be removed before doing a pull request. I think this falls into the latter category.